daprdocs/content/en/developing-applications/debugging/debug-k8s/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Debug Dapr in Kubernetes mode"
+linkTitle: "Kubernetes"
+weight: 2000
```

Comment:
```suggestion
weight: 200
```

Since 'Debugging' was 60, this level should be in the hundreds. Self hosted will eventually be 100